File: 0070-climbing-stairs/0070-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
        if(n==0)
            return 0;
        if(n==1)
            return 1;
        if(n==2)
            return 2;
        int one=1;
        int two=2;
        int total=0;
        int i=2;
        while(i<n)
        {
            total=one+two;
            one=two;
            two=total;
            i++;
        }
        return total;
    }
};

File: 0070-climbing-stairs/0070-climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
    int a=1;
        int b=2;
        int t=0;
                if(n==1)
                    return 1;
        else if(n==2)
            return 2;
        

        
        for(int i=3;i<=n;i++)
        {
            
            t=a+b;
            a=b;
            b=t;
            
        }
        return t;
    }
}